mozanalysis/bootstrap.py

```diff
@@ -0,0 +1,34 @@
+import numpy as np
+from functools import partial
+
+
+def bootstrap(iteration, stat_fn, broadcast_data):
```

Comment:
I'd rename this `_pseudosample` or something, and rename `run_bootstrap` to `bootstrap`

Reply:
Actually, maybe `_resample`

Reply:
Is `bootstrap` a good name? E.g. if we later add a `poisson_bootstrap`, would just plain-old `bootstrap` still make sense?

Reply:
I think if we add poisson, we'd make it a parameter (maybe called `resampling_method`) to a `bootstrap` method